controller/model/controller_manager.go

```diff
@@ -231,12 +231,14 @@ func (self *ControllerManager) PeersConnected(peers []*event.ClusterPeer) {
 	selfAsPeer := self.getCurrentAsClusterPeer()
 	peerFingerprints := ""
 	for _, peer := range peers {
-		fingerprint := nfpem.FingerprintFromCertificate(peer.ServerCert[0])
+		if len(peer.ServerCert) > 0 {
```

Comment:
maybe a nit, but why not just test for nil instead of finding the len ?

Reply:
covers both cases: `nil` and empty